feat: Support customising canvas actions 🎉 (#3364)
excalidraw/excalidraw: src/packages/excalidraw/CHANGELOG.md

### Features

> - Add `UIOptions` prop to support hiding background color picker, clear canvas, export, load, save, save as & dark mode toggle [#3364](https://github.com/excalidraw/excalidraw/pull/3364)

Reviewer: ```suggestion
- Add `UIOptions` prop to customise `canvas actions` which includes customising `background color picker`, `clear canvas`, `export`, `load`, `save`, `save as` & `theme toggle `[#3364](https://github.com/excalidraw/excalidraw/pull/3364)
```
